docs/user-guide/projects/README.md

```diff
@@ -235,11 +232,13 @@ Often you need to see more than one project in your lists and want to create pro
 
 Click on the **Modules** icon with the dots in the header navigation. These are the project overarching modules in OpenProject.![overarching-modules](overarching-modules.png) 
 
-Select **Work packages** from the drop down menu. Now, you will see all work packages in the projects where you have the required [permissions](../..//system-admin-guide/users-permissions/roles-permissions/).
+Select **Work packages** from the drop down menu. Now, you will see all work packages in the projects for which you have the required [permissions](../..//system-admin-guide/users-permissions/roles-permissions/).
 
-You can group by project by clicking in the header of the work package list next to project and select Group by.
+In this project overarching list, you can search, filter, group by, sort, highlight and save views the same way as in the [work package list](../../work-packages/work-package-table-configuration) in each project.
+
+You can group by project by clicking in the header of the work package list next to project and select **Group by**. Collapsing the groups will allow you an **overview of the projects' aggregated milestones** as described [here](../../user-guide/work-packages/work-package-table-configuration/#aggregation-by-project).
```

Comment:
We could change "group by project [...] next to project" to "group by **a** project [...] next to **a** project".

Reply:
I would change it to "group by projects (...) next to PROJECT", as we group by all available projects and the second "project" meant the name of the column. Would that be alright for you?

Reply:
Yes, that's a good idea. Capitalizing PROJECT makes it clear that it is the name of the column.